spec/classes/node_spec.rb

```diff
@@ -66,12 +71,15 @@
             @gluster_params.merge(
               :gluster_replica => 2,
               :gluster_stripe => 3,
-            )
+            ).merge(@redis_params)
```

Comment:
This is getting clunky. I'll try come up with a cleaner way to do it outside this PR.